requirements/extra.txt

```diff
@@ -6,5 +6,6 @@ torchtext>=0.9.*
 omegaconf>=2.0.5
 hydra-core>=1.0.5
 jsonargparse[signatures]>=4.6.0
+docstring-parser<0.14
```

Comment:
I can fix it in jsonargparse and create a patch release so that we can bump jsonargparse instead of adding docstring-parse.

Reply:
Whatever lands faster is good with me :)